cmd/app/app.go

```diff
@@ -47,6 +47,7 @@ func NewCommand(clients *shared.ClientFactory) *cobra.Command {
 		},
 		PostRunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
+			// DEPRECATED(semver:major): remove the "workspace" alias
```

Comment:
📣 Wanting to call this change out! This matches [deprecations](https://github.com/slackapi/slack-cli/blob/main/.github/MAINTAINERS_GUIDE.md#deprecating-features-and-flags) that we might hope to search for in the next set of breaking changes.